microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Analyze selector performance during Recalculate Style events
+description: Use the Selector Stats feature in the Performance tool to understand which of your CSS selectors are taking up the most time during Recalculate Style events.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/09/2023
+---
+# Analyze selector performance during Recalculate Style events
+
+The **Performance** tool highlights long tasks with red triangles to indicate work on the main thread that takes too long and has slow performance.  
```

Comment:
```suggestion
The **Performance** tool highlights long tasks with red triangles to indicate work on the main thread that takes too long and has slow performance:
```
opportunity to use colon to glue flow (equiv of figure captions & references, but low-key)